tsconfig.json

```diff
@@ -11,15 +11,22 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noImplicitReturns": true,
-    "typeRoots": ["node_modules/@types/", "src/types/"],
+    "typeRoots": [
+      "node_modules/@types/",
+      "src/types/"
```

Comment:
Hmm did you install and set Prettier as your default formatter? Or else formatting changes shouldn't happen.

Reply:
No